scripts/run-integration-tests.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
```

Comment:
what about moving these tests into `vitest` with temporary folders created outside the workspace?

Reply:
Would Vitest help us here somehow? I mean are there some Vitest commands available, that make it easier? In the end, it is a bunch of batch scripts which should end successfully.

My initial plan was to have them not running like the normal tests. So really some kind of tests before someone commits or creates a PR.
We could put them into an own vitest config but I'm not sure if it is worth the effort if we don't get any other benefits.